react/hooks/useAppSettings.ts

```diff
@@ -8,7 +8,7 @@ const DEFAULT_PRICES_WITH_TAX = false
 const DEFAULT_USE_SELLER_DEFAULT = false
 const DEFAULT_USE_IMAGES_ARRAY = false
 const DEFAULT_DISABLE_AGGREGATE_OFFER = false
-const DEFAULT_GTIN_VALUE = 'ean'
+const DEFAULT_GTIN_VALUE = 'itemId'
```

Comment:
There’s no need to change the default value, right? The merchant can choose the value that best suits their needs in the app settings in the admin. This way, we avoid the risk of causing side effects for those using the default value.

Reply:
@vsseixaso  actually the default value was itemId, but in the previous release and changes I made it was set default to `ean`